Signal/Images.xcassets/table_ic_new_contact.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "table_ic_new_contact@1x.png",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "filename" : "table_ic_new_contact@2x.png",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "filename" : "table_ic_new_contact@3x.png",
+      "scale" : "3x"
+    }
```

Comment:
I don't like these icons, but I spent a couple hours trying to arrive at something better and couldn't.  Happy to show alternatives if you're interested.